ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUpdaterThread.java

```diff
@@ -625,6 +633,11 @@ public boolean runOneWorkerIteration(
     }
     String cmd = null;
     try {
+      TableName tb = req.tableName;
```

Comment:
If the very first table belongs to DbBeingFailover, it will break the logic for "doWait"

Reply:
add a test where db being failed over is picked up first and later the other db is picked up